shell/lib/asdf.sh

```diff
@@ -69,13 +69,14 @@ asdf_devbase_exec() {
 # .tool-version files in the current directory and all subdirectories.
 asdf_devbase_ensure() {
   readarray -t asdf_entries < <(read_all_asdf_tool_versions)
-  for entry in "${asdf_entries[@]}"; do
+  local need_reshim=0
+  for (( i=0; i < "${#asdf_entries[@]}"; i=(($i + 2)) )); do
```

Comment:
thought: 😬 if only there was a better way to read pairs of words...

Reply:
I mean, if I wasn't lazy I would've turned this into a map instead of an array ("the bashies struct"), but that would've required rewriting more code. :P